patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -55,32 +61,57 @@
  
          if (this.field_70170_p.func_180495_p(p_180429_1_.func_177984_a()).func_177230_c() == Blocks.field_150431_aC)
          {
-@@ -1277,12 +1289,12 @@
-             BlockPos blockpos = new BlockPos(this.field_70165_t, d0, this.field_70161_v);
-             IBlockState iblockstate = this.field_70170_p.func_180495_p(blockpos);
+@@ -1210,7 +1223,7 @@
+         {
+             this.field_70171_ac = false;
+         }
+-
```

Comment:
Try not to add or remove whitespace lines like this one.

Reply:
I might have missed this in latest commit. Will update when I can figure out what that field maps to (I know how to do it, just busy at the moment.)